### PR TITLE
🔨 Updated Contributors List

### PR DESCRIPTION
**Pull request by nomandhoni-cs**
https://github.com/MizouziE/energy-prices-api/issues/23#issue-1243953402

### CODE REVIEW

**Review by MizouziE — CHANGES_REQUESTED** [review]
Hi @nomandhoni-cs , thanks for being the first one to look into using the all-contributors CLI! Normally I do it myself after we've had a few new contributors.

I was planning to add the bot to make this much easier, but haven't yet got around to it, so I have a manual process for doing it which just includes one extra step, if you don't mind just completing that, I'd be very grateful to you for saving me the trouble.

Go have a look in `package.json` and add your details to the list there.

I also just noticed I made a couple of errors in the last two entries, if you could fix the missing `//`'s on those that would be great too.

Thanks!

**Comment by nomandhoni-cs** [comment]
@MizouziE Can please you mention where you made '//' mistake

**Comment by MizouziE** [comment]
> @MizouziE Can please you mention where you made '//' mistake

The last two additions I made to the list, I omitted the slashes from the GitHub URLs after https:

File: package.json [hunk 1]
```diff
@@ -35,7 +35,9 @@
     "Muiz Uvais (https://github.com/MuizU)",
     "Scott Wilson (https://github.com/scottjwilson)",
     "k-puchala (https:github.com/k-puchala)",
-    "Snehashish Ghosh (https:github.com/snehashish-ghosh98)"
+    "Snehashish Ghosh (https:github.com/snehashish-ghosh98)",
```
Comment (MizouziE) [inline package.json:38]: The 'https:github...' is the error I meant. My apologies for not being crystal clear. 

It needs to be 'https://github...' for yours **and the two contributors above you** in the list. 

It's a very minor detail and was my sloppy mistake 😅

**Review by MizouziE — DISMISSED** [review]

File: package.json (hunk 1 above)
Comment (nomandhoni-cs) [reply package.json:38]: @MizouziE No problem, I wasn't able to find this thing

Comment (MizouziE) [reply package.json:38]: Ok, don't worry. I will merge the branch as it is and then make the corrections myself and tag you to show what I meant. Thanks all the same for the contribution!

**Pull request merged**